src/@next/components/atoms/RichTextContent/fixtures/headers.ts

```diff
@@ -0,0 +1,33 @@
+export default `
```

Comment:
Just a suggestion really because this seems to be perfectly fine as it is. I suggest using constant, named exports, not default ones for more explicit naming wherever possible. Here we could either use a file in root component dir named `fixtures.ts` or possibly put it inside a folder `test` to separate it from the component logic itself. It results in easier, more readable imports and we keep exports names consistent. This is not a case here, where we would only use fixtures inside our test file but for general convention it could be a good idea to stick to it.